.github/workflows/release.yml

```
@@ -0,0 +1,146 @@
+      - name: Build sdist and wheel
+        run: |
+          python -m build
+
+      - name: Twine check
+        run: |
+          twine check dist/*
+
+      - name: Extract release notes from CHANGELOG.md
+        run: |
+          python -c "
+import pathlib, re
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)